README.md

```diff
@@ -1,58 +1,86 @@
-- [ ] Links
+- [x] Links
+- [x] [Alternate Assets Object](https://github.com/stac-extensions/alternate-assets?tab=readme-ov-file#alternate-asset-object)
+
+| Field Name     | Type       | Description |
+| -------------- | ---------- | ----------- |
+| `storage:refs` | \[string\] | A property that specifies which schemes in `storage:schemes` may be used to access an Asset or Link. Each value must be one of the keys defined in `storage:schemes`. |
+
+### Storage Scheme Object
 
-| Field Name             | Type      | Description |
-| ---------------------- | --------- | ----------- |
-| storage:platform       | string    | The [cloud provider](#providers) where data is stored |
-| storage:region         | string    | The region where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
-| storage:requester_pays | boolean   | Is the data requester pays or is it data manager/cloud provider pays. *Defaults to false* |
-| storage:tier           | string    | The title for the tier type (as defined by PaaS provider) |
+| Field Name     | Type    | Description |
+| -------------- | ------- | ----------- |
+| platform       | string  | **REQUIRED.** The [cloud provider](#platforms) where data is stored. |
+| region         | string  | The region where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
+| requester_pays | boolean | Is the data requester pays or is it data manager/cloud provider pays. Defaults to `false` |
+| tier           | string  | The title for the tier type (as defined by PaaS provider) |
 
-While these are all valid properties on an Item, they will typically be defined per-asset. If a field applies equally
-to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it should be specified in Item properties.
+The properties `title` and `description` as defined in Common Metadata can be used as well.
 
-### Additional Field Information
+#### Platforms
 
-#### Providers
-Currently this document is arranged to support object storage users of the following PaaS solutions:
+The `platform` field identifies the cloud provider where the data is stored.
+
+There are a couple of pre-defined values for common providers:
 
 - Alibaba Cloud (Aliyun): `ALIBABA`
 - Amazon AWS: `AWS`
 - Microsoft Azure: `AZURE`
 - Google Cloud Platform: `GCP`
 - IBM Cloud: `IBM`
 - Oracle Cloud: `ORACLE`
-- All other PaaS solutions: `OTHER`
 
-The upper-cased values are meant to be used for `storage:platform`.
+All other PaaS solutions must use a unique URL to the service.
+
+In case an `href` contains a non-HTTP URL that is not directly resolvable,
+the `platform` property must identify the host so that the URL can be resolved without further information.
+This is especially useful to provide the endpoint URL for custom S3 providers.
+In this case the `platform` is effectively the endpoint URL.
```

Comment:
Another wild idea. 

Would it be considerable to use `storage` extension to resolve the `docker://` issue that was flagged in https://github.com/stac-extensions/ml-model/issues/9 and https://github.com/stac-extensions/processing/pull/33 ?

I think that could be relevant provided that many of those cloud providers also support container registries, and therefore could need similar platform/region resolution.

Reply:
Not sure. It feels like it mixes concerns and thus warrants a separate extension, but it can re-use the same mechanism, I guess.